Translated start.md and install.md to eu_US (#63)
v2fly/v2fly-github-io: docs/en_US/guide/start.md

# Basics

> You will need to configure the V2Ray after the [installation](install.md). For demostration purpose, we will only introduce the basic configuration. If a complex configuration is required, please refer to [Config Reference](../config/overview.md).

Reviewer: ```suggestion
You will need to configure the V2Ray after the [installation](install.md). For demostration purposes, we will only introduce the basic configuration. If a complex configuration is required, please refer to [Config Reference](../config/overview.md).
```

## Server Side

> For bypassing the censorship, a server with free access to the Internet is required. 

Reviewer: ```suggestion
To circumvent censorship, a server with unfettered access to the Internet is required. 
```

```json
{
    "inbounds": [
        {
            "port": 10086, // Server Listening Port
            "protocol": "vmess",
            "settings": {
                "clients": [
                    {
                        "id": "b831381d-6324-4d53-ad4f-8cda48b30811"
                    }
                ]
            }
        }
    ],
    "outbounds": [
        {
            "protocol": "freedom"
        }
    ]
}
```

> Make sure the `id` and port number matches the client side configuration. 

Reviewer: ```suggestion
Make sure the `id` and port number match the client side configuration. 
```

        {
            "port": 1080, // SOCKS Proxy Port. Configure the browser proxy to use this port.
            "listen": "127.0.0.1",
            "protocol": "socks",
            "settings": {
                "udp": true
            }
        }
    ],
    "outbounds": [
        {
            "protocol": "vmess",
            "settings": {
                "vnext": [
                    {
                        "address": "server", // Server address: Change to your server hostname or ip address.
                        "port": 10086, // Server Listening Port
                        "users": [
                            {
                                "id": "b831381d-6324-4d53-ad4f-8cda48b30811"
                            }
                        ]
                    }
                ]
            }
        },
        {
            "protocol": "freedom",
            "tag": "direct"
        }
    ],
    "routing": {
        "domainStrategy": "IPOnDemand",
        "rules": [
            {
                "type": "field",
                "ip": [
                    "geoip:private"
                ],
                "outboundTag": "direct"
            }
        ]
    }
}
```

> The only part need to be changed in the config above is the server address as mentioned in the comment. These config will proxy your traffic to the server expect for local networks, such as your router.

Reviewer: ```suggestion
The only part need to be changed in the config above is the server address as mentioned in the comment. These config will proxy your traffic to the server except for local networks, such as your router.
```
